templates/homeassistant/spec.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name":"homeassistant",
+  "language":"PYTHON",
+  "prelude": "\"\"\"\nsnips_skill_slot_value = data.get('snips_skill_slot_value')\nservice_data = {'entity_id': 'snips_skill_slot_value' }\nhass.services.call('hass_component', 'hass_component_method', service_data, False)\"\"\"",
```

Comment:
`snips_skill_slot_value` is confusing. Skills are a set of intent and associated actions.
maybe `<intent_slot_name>` is more appropriate so the user at least knows that he has to change this

Reply:
Yep you're right, it's a bit confusing, updated.